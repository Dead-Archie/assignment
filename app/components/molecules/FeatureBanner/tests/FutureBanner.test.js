import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import { FeatureBanner } from '../FeatureBanner';

const sampleProps = {
  className: 'withStyles-sc-4wveuc-0 lgzevM',
  heading: 'Heading Test',
  subheading: 'Sub Heading Test',
};
const setUp = props => {
  return shallow(<FeatureBanner {...props} />);
};

describe('<FeatureBanner />', () => {
  let FeatureBannerComponent;
  beforeEach(() => {
    FeatureBannerComponent = setUp(sampleProps);
  });

  test('should render correctly', () => {
    expect(FeatureBannerComponent).toMatchSnapshot();
  });
});
