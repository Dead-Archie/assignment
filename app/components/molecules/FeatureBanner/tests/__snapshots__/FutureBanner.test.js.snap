// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeatureBanner /> should render correctly 1`] = `
<article
  className="lgzevM"
>
  <div
    className="inner"
  >
    <h1>
      Heading Test
    </h1>
    <div
      className="subheading"
    >
      Sub Heading Test
    </div>
  </div>
</article>
`;
